cmd/main.go

```diff
@@ -59,7 +66,7 @@ func main() {
 		},
 	}
 
-	ctx := opio.WithInterruptBlocker(context.Background())
+	ctx := ctxinterrupt.WithSignalWaiterMain(context.Background())
```

Comment:
needed to be updated based on refactor done in https://github.com/ethereum-optimism/optimism/pull/11511